build.gradle

```diff
@@ -0,0 +1,49 @@
+plugins {
```

Comment:
there is no need to track these files (`build.gradle`, `checkstyle`, `suppressions` and can be added to gitignore